src/main/java/com/coveros/selenified/utilities/Listener.java

```diff
@@ -142,7 +140,14 @@ public void onTestFailure(ITestResult result) {
     @Override
     public void onTestSkipped(ITestResult result) {
         super.onTestSkipped(result);
-        recordResult(result);
+        //cleanup unneeded files
+        Reporter reporter = (Reporter) result.getAttribute(REPORTER);
+        if (reporter != null) {
+            new File(reporter.getDirectory(), reporter.getFileName() + ".html").delete();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use "java.nio.Files#delete" here for better messages on error conditions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS4042)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Do something with the "boolean" value returned by "delete". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS899)